src/ui/units/editor/store/reducers/editor/editor.js

```diff
@@ -384,6 +385,10 @@ export const getScriptsValues = (state) => state.editor.scriptsValues;
 export const getIsScriptsChanged = createSelector(
     [getEntry, getScriptsValues],
     (entry, scriptsValues) => {
+        if (entry.meta.is_sandbox_version_changed) {
```

Comment:
`entry.meta?.is_sandbox_version_changed`

`meta` may not be defined

Reply:
Yep, fixed it